src/sentry/sentry_metrics/indexer/mock.py

```diff
@@ -1,41 +1,44 @@
 from typing import Optional
 
-from sentry.models import Organization
+from django.conf import settings
+
+from sentry.utils.redis import redis_clusters
 
 from .base import StringIndexer, UseCase
 
-_STRINGS = {
```

Comment:
Some unit tests actually rely on these mock integer values (e.g. [here](https://github.com/getsentry/sentry/blob/e59971700f9f1c1778030cc3944805d845a79e78/tests/sentry/snuba/test_metrics.py#L106)), so it might make sense to keep the `MockIndexer` as it was, and put the redis-based indexer in a new subclass of `StringIndexer`.